tooling/v8-snapshot/src/setup/force-no-rewrite.ts

```diff
@@ -1,76 +1,54 @@
+/**
+ * These modules are force no rewritten because they are rewritten in a way that
+ * breaks the snapshot. When run through the snapshot generator, these strings
+ * will be compared to the end of each file's path and if they match, the given
+ * file will be marked as force no rewritten. For files in the project, we should
+ * include the start of the path (e.g. packages/https-proxy/lib/ca.js). For files
+ * in node_modules, we should exclude the start of the path (e.g.
+ * force-no-rewrite/index.js) so that dependencies that are hoisted in other
+ * node_modules directories are also not rewritten.
+ */
```

Comment:
```suggestion
/**
 * These modules are force no rewritten because they are rewritten in a way that
 * breaks the snapshot but is not detected automatically by the snapshot builder. 
 * When run through the snapshot generator, these strings
 * will be compared to the file's path and if they match, the given
 * file will be marked as force no rewritten. If we want to match the full path, we
 * should include the full relative path with respect to the project base (e.g. 
 * packages/https-proxy/lib/ca.js). For files where we want to match multiple hoisted
 * locations, we should specify the dependency starting with `*/` (e.g.
 * */node_modules/signal-exit/index.js)
 */
```